packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -86,11 +86,27 @@ export class NativeService {
   ): Promise<Connection> =>
     await ParentHandshake(messenger, this.localMethods, numbers.maxAttempts)
 
-  public tryEmit = async (event: NativeEvent): Promise<void> => {
+  public tryEmit = async (
+    event: NativeEvent,
+    componentId: string
+  ): Promise<void> => {
     if (!this.isNativeEvent(event)) return
 
     const parsedEvent = this.parseNativeEvent(event)
 
+    if (parsedEvent.methodName === 'setFlagshipUI' && parsedEvent.args) {
+      // @ts-expect-error we know that `setFlagshipUI` args are in an array
```

Comment:
Not sure to understand, if args is still a string at this point, accessing [0] would be a letter value? If it can be an array, we should update the `PostMeMessage` interface that is returned by `parseNativeEvent()`